test/HydraDelegation/VestedDelegation.test.ts

```diff
@@ -1715,15 +1800,97 @@ export function RunVestedDelegationTests(): void {
       });
     });
 
+    describe("Liquid Debt", async function () {
+      it("should give negative liquid debt when vesting", async function () {
+        const { hydraDelegation, vestManager } = await loadFixture(this.fixtures.vestedDelegationFixture);
+
+        const liquidDebt = await hydraDelegation.liquidityDebts(vestManager.address);
+
+        expect(liquidDebt).to.be.lt(0);
+      });
+
+      it("Should clear all debt and take all Lydras on cutting the whole position", async function () {
+        const { systemHydraChain, hydraStaking, hydraDelegation, vestManager, delegatedValidator, vestManagerOwner } =
+          await loadFixture(this.fixtures.vestedDelegationFixture);
+
+        // commit epochs to distribute rewards
+        await commitEpochs(systemHydraChain, hydraStaking, [delegatedValidator], 5, this.epochSize, WEEK);
+
+        const liquidDebtBefore = await hydraDelegation.liquidityDebts(vestManager.address);
+
+        await vestManager.connect(vestManagerOwner).cutVestedDelegatePosition(delegatedValidator.address, 100);
+
+        const liquidDebtAfter = await hydraDelegation.liquidityDebts(vestManager.address);
+
+        expect(liquidDebtAfter).to.be.eq(liquidDebtBefore.add(100));
+        expect(liquidDebtAfter).to.be.gt(liquidDebtBefore);
+        expect(liquidDebtAfter).to.be.lt(0);
+      });
+
+      it("When cutting a position, the liquid debt should be changed properly (above), and we provide 0 Lydra", async function () {
+        const {
+          systemHydraChain,
+          hydraStaking,
+          hydraDelegation,
+          vestManager,
+          delegatedValidator,
+          vestManagerOwner,
+          liquidToken,
+        } = await loadFixture(this.fixtures.vestedDelegationFixture);
+
+        // commit epochs to distribute rewards
+        await commitEpochs(systemHydraChain, hydraStaking, [delegatedValidator], 5, this.epochSize, WEEK);
+
+        const liquidDebtBefore = await hydraDelegation.liquidityDebts(vestManager.address);
+
+        await liquidToken.connect(vestManagerOwner).approve(vestManager.address, this.minDelegation.mul(100));
+        // cut the entire position without providing any Lydra, because the liquid debt is more than the position for the manager
+        await vestManager
+          .connect(vestManagerOwner)
+          .cutVestedDelegatePosition(delegatedValidator.address, this.minDelegation.mul(2));
+
+        const liquidDebtAfter = await hydraDelegation.liquidityDebts(vestManager.address);
+
+        expect(await liquidToken.balanceOf(vestManagerOwner.address)).to.be.eq(0);
+        expect(liquidDebtAfter).to.be.gt(liquidDebtBefore);
+        expect(liquidDebtAfter).to.be.eq(0);
+      });
+
+      it("When opening a big position (more than the liquid debt), another position with same manager will handle liquid debt properly", async function () {
```

Comment:
Use specific expected values for the test to ensure the math is right, not only the +/-.